utilities.h

```diff
@@ -153,11 +169,11 @@ std::string ToLower(std::string const &text);
 std::string ToUpper(std::string const &text);
 
 // replaces polish letters with basic ascii
-void win1250_to_ascii( std::string &Input );
-// TODO: unify with win1250_to_ascii()
+void remove_accents( std::string &Input );
+// TODO: unify with remove_accents()
 std::string Bezogonkow( std::string Input, bool const Underscorestospaces = false );
```

Comment:
`Bezogonkow` assumes win1250 input, and I'm not really sure that this holds true when files are in UTF8. All uses of this should be eliminated. Additional functions utf8_to_ascii and utf8_to_file will probably be needed. Take note that when deciding to use file_to_utf8 during deserialization file_to_utf8 will need to be used during serialization.